### DISCUSSION TITLE
New Event: AFTER_UPDATE_YAML_FILES

### DISCUSSION THREAD

**Discussion by mijewe** [craftcms/cms#14365] [Ideas] [answered]
My client needs to edit the project config on the production site, and I was thinking that I could run some Git commands automatically after any Project Config yaml files get updated to keep everything in sync.

However, there's no event that gets fired after a successful `ProjectConfig->updateYamlFiles()`

Hooking into the existing `ProjectConfig::EVENT_ADD_ITEM` events doesn't quite cut it, as the actual files won't have been written by then.

**Comment by brandonkelly** [comment, marked as answer]
Just added an `EVENT_AFTER_WRITE_YAML_FILES` event for Craft 4.8 (879d63866058cfee4855f4ee8522fdc136e235c6).

To update to the `4.8` branch now, change your `craftcms/cms` requirement to `4.8.x-dev as 4.8.0-alpha` in composer.json, then run `composer update`.

**Reply by brandonkelly** [reply]
Craft 4.8.0 is out now with that event.
